code/game/objects/items/food/soup.dm

```diff
@@ -44,6 +45,7 @@
 		/datum/reagent/consumable/nutriment = 6,
 		/datum/reagent/drug/mushroomhallucinogen = 6,
 		/datum/reagent/consumable/nutriment/vitamin = 5,
+		trash_type = /obj/item/reagent_containers/cup/bowl
 	)
 	tastes = list("jelly" = 1, "mushroom" = 1)
```

Comment:
this doesn't belong in the list, it's a variable not a list entry, so you want it after the ) here